src/main/java/frc/bionic/swerve/command/DriveWithJoystick.java

```diff
@@ -85,6 +86,6 @@ public void execute() {
     SmartDashboard.putNumber("ySpeed", ySpeed);
     SmartDashboard.putNumber("rot",    rot);
 
-    drivetrain.drive(xSpeed, ySpeed, rot);
+    drivetrain.drive(xSpeed, ySpeed, joystick.getRawButton(2) ? rot : 0);
```

Comment:
I think this needs a comment stating (1) why is this button press necessary, and (2) what is raw button 2